templates/inspec/examples/google_spanner_instance/google_spanner_instances.erb

```diff
@@ -0,0 +1,6 @@
+<% gcp_project_id = "#{external_attribute('gcp_project_id', doc_generation)}" -%>
+<% spannerinstance = grab_attributes['spannerinstance'] -%>
+
+describe google_spanner_instances(project: <%= gcp_project_id -%>, config: <%= doc_generation ? "'#{spannerinstance['config']}'" : "spannerinstance['config']" -%>) do
+  its('configs') { should include <%= doc_generation ? "'#{spannerinstance['config']}'" : "spannerinstance['config']" -%> }
```

Comment:
Similar to the issues with match above, either construct the name here to compare with or do something like this:
```
describe.one do
  google_spanner_instances(project: <%= gcp_project_id -%>, config: <%= doc_generation ? "'#{spannerinstance['config']}'" : "spannerinstance['config']" -%>).configs.each do |config|
    describe config do
      it { should match <%= doc_generation ? "'#{spannerinstance['config']}'" : "spannerinstance['config']" -%> }
    end
  end
end
```
Similar to https://github.com/GoogleCloudPlatform/magic-modules/blob/c1cebaf1374b73ad7ee5cf660ec29fa2cfe5d6d8/templates/inspec/examples/google_bigquery_dataset/google_bigquery_dataset.erb#L13

Reply:
this looks better. So adding it